SpeedyContactAdd WatchKit Extension/Controller/InterfaceController.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  InterfaceController.swift
+//  SpeedyContactAdd WatchKit Extension
+//
+//  Created by Aaron Haughton on 7/16/16.
+//  Copyright © 2016 AlphaGradeINC. All rights reserved.
+//
+
+import CoreLocation
+import UIKit
+import WatchConnectivity
+import WatchKit
+
+class InterfaceController: WKInterfaceController, WCSessionDelegate {
+    
+    override func awake(withContext context: Any?) {
+            if(WCSession.isSupported()){
+            self.session = WCSession.default
+            session.delegate = self
+            self.session.activate()
+        }
+    }
+    override func willActivate() {
+        // This method is called when watch view controller is about to be visible to user
+        super.willActivate()
+
+    }
+
+    func session(_ session: WCSession,
+                 activationDidCompleteWith activationState: WCSessionActivationState,
+                 error: Error?){
+        if let error = error {
+            print("There was an error: \(error)")
+            return
+        }
+  
+    }
+        
+    
+    func sessionReachabilityDidChange(_ session: WCSession) {
+        
+    }
+    
+    @IBOutlet var lblPhoneNumber: WKInterfaceLabel!
+    @IBOutlet weak var lblReName: WKInterfaceLabel!
+    
+    var session:WCSession!
+    
+    var contacts: [Contacts] = []
+    var numberStore = String()
+    var recipNameString = String()
+    var fullNameString = "fullNameString"
+    var numberString = "numberString"
+    var timer = Timer()
+    let contactsDefault = UserDefaults.standard
+    let checkIfUserDefaultExist:((String) -> Bool) = { key in
+        return UserDefaults.standard.object(forKey: key) != nil
+    }
+    
+    func numberAdd(myCharacter:String) {
+        numberStore += myCharacter
+        lblPhoneNumber.setText(numberStore)
+    }
+    func clearItems() {
+        numberStore = ""
+        recipNameString = ""
+        lblReName.setText("Contact Name")
+        lblPhoneNumber.setText("")
+    }
+    
+    //Mark: Keypad that addes numbers to phone number
+    @IBAction func btnNumber1Action(){
+        numberAdd(myCharacter: "1")
+    }
+    @IBAction func btnNumber2Action() {
+        numberAdd(myCharacter: "2")
+    }
+    @IBAction func btnNumber3Action() {
+        numberAdd(myCharacter: "3")
+    }
+    @IBAction func btnNumber4Action() {
+        numberAdd(myCharacter: "4")
+    }
+    @IBAction func btnNumber5Action() {
+        numberAdd(myCharacter: "5")
+    }
+    @IBAction func btnNumber6Action() {
+        numberAdd(myCharacter: "6")
+    }
+    @IBAction func btnNumber7Action() {
+        numberAdd(myCharacter: "7")
+    }
+    @IBAction func btnNumber8Action() {
+        numberAdd(myCharacter: "8")
+    }
+    @IBAction func btnNumber9Action() {
+        numberAdd(myCharacter: "9")
+    }
+    @IBAction func btnNumber0Action() {
+        numberAdd(myCharacter: "0")
+    }
+    @IBAction func btnBackspaceAction() {
+        if numberStore != "" {
```

Comment:
```suggestion
        if !numberStore.isEmpty {
```
(Faster for performance)